spring-boot/config-server/src/main/java/eu/coatrack/admin/server/service/filter/CheckIfUrlAndCredentialsFitWithEachOtherFilter.java

```diff
@@ -42,24 +42,19 @@
     private final String CREDENTIAL_URI = "/credentials";
 
     @Override
-    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-        if (request instanceof HttpServletRequest
+    protected void doFilterInternal(HttpServletRequest servletRequest, HttpServletResponse servletResponse, FilterChain filterChain) throws ServletException, IOException {
```

Comment:
As mentioned before: the method is too long and has too many levels of indentation. It should be split up to multiple smaller methods.

Reply:
I would not do that in this pull request, because:

1. this method is not really important for the PR, just a minor improvement was done
2. if we re-factor this method and accidentally introduce an error, it will be hard to track down as it has nothing to do with the merged PR

Maybe a separate, low-priority issue makes sense.